Flaky test: TestMarsharl

```log
=== RUN   TestMarsharl
    object_storage_test.go:619: PUT failed: create /.hello.tmp.679478056050146678: addBlock call failed with ERROR_APPLICATION (java.io.IOException)
--- FAIL: TestMarsharl (0.01s)
```

https://github.com/juicedata/juicefs/actions/runs/3538221106/jobs/5938870538 also refer to https://github.com/juicedata/juicefs/issues/2576

--- a/.github/scripts/setup-hdfs.sh
+++ b/.github/scripts/setup-hdfs.sh
@@ -20,7 +20,7 @@ sudo apt-get update
 sudo apt-get install openjdk-8-jdk -y
 
 HADOOP_VERSION="2.10.1"
-wget https://dlcdn.apache.org/hadoop/common/hadoop-2.10.1/hadoop-2.10.1.tar.gz
+wget -q https://dlcdn.apache.org/hadoop/common/hadoop-2.10.1/hadoop-2.10.1.tar.gz
 mkdir ~/app
 tar -zxf hadoop-${HADOOP_VERSION}.tar.gz -C ~/app
 

--- a/.github/scripts/setup-hdfs.sh
+++ b/.github/scripts/setup-hdfs.sh
@@ -80,7 +80,7 @@ jps
 
 echo "hello world" > /tmp/testfile
 cd ~/app/hadoop-${HADOOP_VERSION}/bin
-./hdfs dfs put /tmp/testfile /
-./hdfs dfs rm /testfile
+./hdfs dfs -put /tmp/testfile /
+./hdfs dfs -rm /testfile
 
 echo "hdfs started successfully"

--- a/.github/scripts/setup-hdfs.sh
+++ b/.github/scripts/setup-hdfs.sh
@@ -78,4 +78,9 @@ cd ~/app/hadoop-${HADOOP_VERSION}/sbin
 
 jps
 
+echo "hello world" > /tmp/testfile
+cd ~/app/hadoop-${HADOOP_VERSION}/bin
+./hdfs dfs put /tmp/testfile /
+./hdfs dfs rm /testfile
+
 echo "hdfs started successfully"

--- a/.github/workflows/mutate-test.yml
+++ b/.github/workflows/mutate-test.yml
@@ -137,9 +137,9 @@ jobs:
           docker run -d --name sftp -p 2222:22  juicedata/ci-sftp
           make
           sudo make -C fstests setup
-          chmod +x travis-setup-hdfs.sh
+          chmod +x .github/scripts/setup-hdfs.sh
           sudo lsof -i :8020 || true
-          ./travis-setup-hdfs.sh
+          ./.github/scripts/setup-hdfs.sh
           for i in {2222,3306,5432,8020,9000,9007,4500} ; do echo "lsof port:"$i;sudo lsof -i :$i;if [ $? != 0 ];then sleep 5; else continue; fi;sudo lsof -i :$i; if [ $? != 0 ];then echo "service not ready, port:"$i; exit 1;fi; done
           sudo lsof -i :2379;if [ $? != 0 ];then echo "tikv is not ready";cat output.log;exit 1; fi
 

--- a/.github/workflows/unittests.yml
+++ b/.github/workflows/unittests.yml
@@ -94,9 +94,9 @@ jobs:
           docker run -d --name sftp -p 2222:22  juicedata/ci-sftp
           make
           sudo make -C fstests setup
-          chmod +x travis-setup-hdfs.sh
+          chmod +x .github/scripts/setup-hdfs.sh
           sudo lsof -i :8020 || true
-          ./travis-setup-hdfs.sh
+          ./.github/scripts/setup-hdfs.sh
           for i in {2222,3306,5432,8020,9000,9007,4500} ; do echo "lsof port:"$i;sudo lsof -i :$i;if [ $? != 0 ];then sleep 5; else continue; fi;sudo lsof -i :$i; if [ $? != 0 ];then echo "service not ready, port:"$i; exit 1;fi; done
           sudo lsof -i :2379;if [ $? != 0 ];then echo "tikv is not ready";cat output.log;exit 1; fi
 

